src/extended-schema/extended-schema.ts

```diff
@@ -11,7 +11,7 @@ export interface LinkConfig {
     batchMode: boolean
     keyField?: string
     ignore?: boolean
-
+    oneToMany?: boolean
```

Comment:
Maybe we can start documenting fields, something along the lines of *Set to true to link this field to many target objects. The target field should accept one key as argument and return a list of items. Can not be combined with batchMode.*

Reply:
I added JSDoc comments to all the properties of the linkConfig interface (to the best of my understanding).

Reply:
Awesome!